src/ad-tracking/scroll-tracker.ts

```diff
@@ -0,0 +1,63 @@
+import { context, events, eventService, scrollSpeedCalculator } from '@ad-engine/core';
+
+class ScrollTracker {
+	scrollSpeedTrackingStarted: boolean;
+	timer: NodeJS.Timer;
+
+	constructor() {
+		this.scrollSpeedTrackingStarted = false;
+		this.timer = null;
+	}
+
+	/**
+	 * Init scroll speed tracking when enabled
+	 */
+	initScrollSpeedTracking(applicationAreaClass: string): void {
+		if (!context.get('options.scrollSpeedTracking')) {
+			return;
+		}
+
+		const applicationArea = document.getElementsByClassName(applicationAreaClass)[0];
+
+		this.scrollSpeedTrackingStarted = false;
+		applicationArea.addEventListener('touchstart', () => this.trackScrollSpeedToDW());
```

Comment:
It may throw an error (`applicationArea` can be undefined)

Reply:
What about SPA? Would below code solve the problem?
```suggestion
		applicationArea.addEventListener('touchstart', () => this.trackScrollSpeedToDW(), { once: true });
```

Reply:
I just found reset method :P. Anyway "once" would be nice here too